src/sql/QueryEditor/FillValueSelect.tsx

```diff
@@ -43,7 +43,7 @@ export function FillValueSelect<TQuery extends DataQuery & Record<string, any>>(
               // Keep the fillMode.value in case FillValueOptions.Value mode is selected back
               fillMode: { ...props.query.fillMode, mode: value },
             });
-            props.onRunQuery();
+            props?.onRunQuery?.();
```

Comment:
@iwysiu you only changed one, but all the calls should be `props.onRunQuery?` :sweat_smile:

Reply:
oops, i did not realize there were multiple